### DISCUSSION TITLE
Нормальные маски подстекольников

### DISCUSSION THREAD

**Discussion by fedotdev** [metrostroi-repo/MetrostroiAddon#213] [Ideas]
Предлагаю заменить **подстекольники**, которые сейчас есть в деве на эти, которые я недавно делал для аддона. Эти более правдоподобные, а что сейчас это просто наклейки какие-то. 
![image](https://user-images.githubusercontent.com/44346544/117513105-afd8a500-af99-11eb-8521-0fc8f0d7e415.png)
![image](https://user-images.githubusercontent.com/44346544/117513137-c5e66580-af99-11eb-8aa6-e3b978269cc9.png)
![image](https://user-images.githubusercontent.com/44346544/117513647-d5b27980-af9a-11eb-9295-70561532bad1.png)
![image](https://user-images.githubusercontent.com/44346544/117513748-0db9bc80-af9b-11eb-9289-b477c8d684a5.png)

**Comment by cyber-dream** [comment]
Привет, спасибо. Возьмём в работу чуть позже, после фиксов меньшего масштаба, они у нас по приоритету первыми идут
